blueocean-jira/pom.xml

```diff
@@ -38,10 +38,6 @@
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>apache-httpcomponents-client-4-api</artifactId>
         </dependency>
-        <dependency>
-            <groupId>com.google.guava</groupId>
-            <artifactId>guava</artifactId>
-        </dependency>
```

Comment:
was never a good thing - so nuke this from orbit with extreme prejudice.

the tests where not updated - but leaving the dependency here would promote the dep to test scope which would likely break things like `hpi:run` as Jenkins would not have `guava` when it should have.